index.js

```diff
@@ -11,13 +13,134 @@ const FILE = 'pokedex.json';
 
 // Init express app
 const app = express();
+console.log("done creating app");
+
+// tell app to use the module
+app.use(express.json());
+app.use(express.urlencoded({
+    extended: true
+}));
+
+// method-override
+const methodOverride = require('method-override');
+app.use(methodOverride('_method'));
+
+// this line below, sets a layout look to your express project
+const reactEngine = require('express-react-views').createEngine();
+app.engine('jsx', reactEngine);
+
+// this tells express where to look for the view files
+app.set('views', __dirname + '/views');
+
+// this line sets react to be the default view engine
+app.set('view engine', 'jsx');
+
+// app.get('/', (req, res) => {
+//   // running this will let express to run home.handlebars file in your views folder
+//   res.render('home')
+// })
+
 
 /**
  * ===================================
  * Routes
  * ===================================
  */
 
+app.get('/', (request, response)=> {
+    const nameArr = [];
+    jsonfile.readFile(FILE, (err, obj) => {
+        for (i=0; obj['pokemon'].length; i++) {
+            nameArr.push(obj['pokemon'][i]['name']);
+        }
+        const nameArrJoin = nameArr.join(", ");
+        response.send('<html><h1>Welcome to the online Pokedex!</h1><body>' + nameArrJoin + '<br>' +
+            '<form method="GET" action="/sortby = name"><input type="submit" value="Sort"></form></body></html>' )
+    })
+    console.log(nameArr);
+})
+
```

Comment:
```suggestion
        function sortPokemonByName(pokemonA, pokemonB) {
            var nameA = pokemonA.name.toLowerCase();
            var nameB = pokemonB.name.toLowerCase();
            if (nameA < nameB) {
              return -1;
            }
            if (nameA > nameB) {
              return 1;
            }
            return 0;
          }
          pokedex["pokemon"].sort(sortPokemonByName);
```
sample sort function you can try using later when you are looking to sort an array of objects